### ISSUE TITLE
Issue information are not being read for very first time

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#93]
When you open Accessibility Checker balloon for the first time, it will not read issue information (position, title, descr).

This might have something to do with with poor handling of `aria-live` attribute.
1. Run JAWS + FireFox.
2. Open any sample with Accessibility Checker. (i.e. [plugins/a11ychecker/dev/sample.html](http://ckeditor.dev/plugins/a11ychecker/dev/sample.html))
3. Focus the editor.
4. Enable AC with a hotkey.

**Expected result:**
JAWS should read: (...) (dialog title) dialog, (button focused) button, Issue X of Y (error), (issue title), (issue descr)

**Current result:**
JAWS should read: (...) (dialog title) dialog, (button focused) button

So it basicaly doesn't read elements with `aria-live` attribute.

This ticket is a follow-up of #22
